meteor-app/imports/ui/pages/workspace/component.js

```diff
@@ -214,15 +222,13 @@ export default class WorkspacePage extends React.Component {
                   </div>
                   <div className="layer-opacity-row">
                     <label>Opacity: </label>
-                    <input
+                    <Slider
                       className="input-slider--layer-opacity"
-                      type="range"
-                      min="0"
-                      max="255"
-                      step="1"
+                      min={0}
+                      max={255}
                       value={layer.opacity * 255}
                       data-layer-index={layerIndex}
-                      onChange={this._bound_layerOpacityOnChange}
+                      onChange={newValue => this._bound_layerOpacityOnChange(newValue, layerIndex)}
```

Comment:
This strategy is less ideal as it creates a new function **every time** the `render` function is called.